### PR TITLE
Switch to GitHub actions

### PR DESCRIPTION
**Pull request by bcomnes**
Switch to GitHub actions and test other OSs as well as LTS node.

### CODE REVIEW

**Comment by ungoldman** [comment]
fine by me 👍

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [8, 10, lts/*]
```
Comment (ungoldman) [inline .github/workflows/tests.yml:13]: I think we can safely drop 8 and 10 considering they've been off LTS for a long time. https://nodejs.org/en/about/releases/

**Review by ungoldman — APPROVED** [review]
thanks for continuing to maintain this @bcomnes

File: .github/workflows/tests.yml (hunk 1 above)
Comment (bcomnes) [reply .github/workflows/tests.yml:13]: Ok sounds good, want me to do a major release bumping the node version?

**Comment by bcomnes** [comment]
No problem! It's finding some use at @socketdev and decided to take a peek inside again.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (bcomnes) [reply .github/workflows/tests.yml:13]: Merged as is. Let me know how you want that handled.

**Pull request merged**